test/prepare_translations.py

```diff
@@ -0,0 +1,53 @@
+"""this script should run every time the contents of the locale folder change
```

Comment:
this script is not needed anymore, was only used to generate initial translation json files